configs/variables

```diff
@@ -13,9 +13,9 @@ FEDORA_HOME="/usr/local/fedora"
 FFMPEG_VERSION=1.1.4
 
 FITS_HOME="/usr/local/fits"
-FITS_VERSION=0.10.1
+FITS_VERSION=1.1.1
 
-SOLR_HOME="/usr/local/solr"
-SOLR_VERSION=4.2.0
+SOLR_HOME="/usr/local/fedora/solr"
```

Comment:
@lutaylor I get it, those properties are set for the build in Solr. I would suggest you go for sed. It's good practice for people poking around in the machine to have the different services in their own directories, but mostly in the same place, they were available few months ago.